hack/test_e2e.sh

```diff
@@ -519,7 +519,12 @@ function main {
 
     install_kpng
 
-    run_tests
+    if [ "${devel_mode}" = true ] ; then
+	pass_message "KPNG development env created, use kubectl in the namespace ${NAMESPACE} to manage it."
+	exit 0
+    else
```

Comment:
Is `else` block really needed? You're doing an `exit 0` anyway

Reply:
A good call. :) Let me fix this.